lib/puppetserver/ca/puppet_config.rb

```diff
@@ -1,9 +1,9 @@
 
 module Puppetserver
   module Ca
-    # Provides an interface for asking for Puppet[ Server] settings w/o loading
-    # either Puppet or Puppet Server. Includes a simple ini parser that will
-    # ignore Puppet's more complicated conventions.
+    # Provides an interface for asking for Puppet settings w/o loading
+    # Puppet. Includes a simple ini parser that will ignore Puppet's
+    # more complicated conventions.
```

Comment:
I didn't see a lot of benefit to trying to combine this with the puppetserver version, or to use inheritance to relate them, but I'm open to ideas if someone thinks there'd be a better way to organize the two config classes.